DEVELOPMENT.md

```diff
@@ -114,17 +114,31 @@ Tekton Dashboard does not involve any custom resource definitions, we only inter
 You can stand up a version of the dashboard on-cluster (to your
 `kubectl config current-context`):
 
+First install and build the npm project.
+
+```shell
+npm install
+```
+
+There is a dedicated npm job for ko builds
+
+```shell
+npm run build_ko
+```
+
+This will build the static resources and add them to the `kodata` directory.
+
 ```shell
 ko apply -f config/
 ```
 
 ## Access the dashboard
 
-To access the backend API:
+To access the dashboard:
 
 `kubectl port-forward $(kubectl get pod -l app=tekton-dashboard -o name) 9097:9097`
 
-Note that we have a big TODO which is to link up the frontend to the backend and to document how to build and run the frontend for yourself.
+The web UI can be accessed at http://localhost:9097
```

Comment:
404 page not found 😞 I see the Go code looks ok, I don't get my logging.Log.Info("Adding web api") message

Reply:
^^ above was from my own directory structure being wrong, now I get:

```
{"level":"info","msg":"Adding web api"}
{"level":"info","msg":"Adding v1, and API for pipelines"}
{"level":"info","msg":"Adding API for websocket"}
{"level":"info","msg":"Adding API for health"}
{"level":"info","msg":"Adding API for readiness"}
{"level":"info","msg":"Searching for extensions in the namespace "}
{"level":"info","msg":"Adding API for extensions"}
{"level":"debug","msg":"Extension: []"}
panic: http: multiple registrations for /

goroutine 1 [running]:
net/http.(*ServeMux).Handle(0xc00036c390, 0x1223e06, 0x1, 0x13726e0, 0xc0004a3ec0)
        /usr/local/go/src/net/http/server.go:2377 +0x221
net/http.(*ServeMux).HandleFunc(0xc00036c390, 0x1223e06, 0x1, 0xc0004a3ec0)
        /usr/local/go/src/net/http/server.go:2395 +0x5a
```